hs_core/views/utils.py

```diff
@@ -124,7 +124,11 @@ def validate_url(url):
 
     # validate url is valid, i.e., can be opened
     try:
-        urlopen(url)
+        # have to add a User-Agent header and pass in a Request to urlopen to test 
+        # whether a url can be resolved since some valid website URLs block web 
+        # spiders/bots, which raises a 403 Forbidden HTTPError
+        url_req = Request(url, headers={'User-Agent': 'Mozilla/5.0'})
```

Comment:
Would this version number "5.0" get deprecated and cause issues as time goes?

Reply:
@zhiyuli I have the same concern, but setting this User-Agent is the only one I can get from googling. If you can find a better one, please let me know.